src/txn/txn_rollback_to_stable.c

```diff
@@ -232,8 +232,10 @@ __rollback_row_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW
          * we can skip it.
          */
         if (__wt_txn_visible_all(
-              session, cbt->upd_value->tw.stop_txn, cbt->upd_value->tw.durable_stop_ts))
+              session, cbt->upd_value->tw.stop_txn, cbt->upd_value->tw.durable_stop_ts)) {
```

Comment:
You can use the helper function __wt_txn_tw_stop_visible_all for this.

Reply:
Thanks. Good pick up.

52fe369